lib/matplotlib/collections.py

```diff
@@ -27,54 +27,41 @@
     "linewidth": ["linewidths", "lw"],
 })
 class Collection(artist.Artist, cm.ScalarMappable):
+    r"""
+    Base class for Collections. Must be subclassed to be usable.
+
+    A Collection represents a sequence of `~.patches.Patch`\s that can be drawn
+    more efficiently together than individually. For example, when a single
+    path is being drawn repeatedly at different offsets, the renderer can
+    typically execute a ``draw_marker()`` call much more efficiently than a
+    series of repeated calls to ``draw_path()`` with the offsets put in
+    one-by-one.
+
+    Most properties of a collection can be configured per-element. Therefore,
+    Collections have "plural" versions of many of the properties of a
+    `~.patches.Patch` (e.g. `.Collection.get_paths` instead of
+    `~.patches.Patch.get_path`). Exceptions are the *zorder*, *hatch*,
+    *pickradius*, *capstyle* and *joinstyle* properties, which can only be set
+    globally for the whole collection.
+
+    Besides these exceptions, all properties can be specified as single values
+    (applying to all elements) or sequences of values. The property of the
+    ``i``th element of the collection is::
```

Comment:
```suggestion
    ``i``-th element of the collection is::
```
https://ell.stackexchange.com/a/77550

I assume that works, but t.b.c. If not, you'll have to escape with a hyphen.

Reply:
good catch! doesn't work actually, but I prefer `i`th to `i`-th in instances where the `i` (as in this case) has a distinct font. Added a slash to escape the "th".